src/components/Professors/ProfessorsTable.tsx

```diff
@@ -81,6 +93,7 @@ const ProfessorsTable = ({ professors, openDetails }) => {
                             name: `${firstName} ${lastName}`,
                             type: type,
                             status: <CompleteStatusBadge complete={complete} />,
+                            is_peng: isPeng ? "Yes" : "No",
```

Comment:
Maybe "PEng" or "" (blank) if not? What do you think?

Reply:
Thanks! I'll try it

Reply:
![image](https://user-images.githubusercontent.com/91636336/180055888-3fecf314-f69d-4892-b732-5a21cd8aa10a.png)
Yeah I think it looks better with no "No"